src/backend/efiling-worker/src/test/java/ca/bc/gov/open/jag/efilingworker/EfilingMessageConsumerTest.java

```diff
@@ -0,0 +1,13 @@
+package ca.bc.gov.open.jag.efilingworker;
+
+import ca.bc.gov.open.jag.efilinglookupclient.MockCSOLookupServiceImpl;
+import ca.bc.gov.open.jag.efilingworker.service.MockDocumentStoreServiceImpl;
+import org.junit.jupiter.api.Test;
+
+public class EfilingMessageConsumerTest {
```

Comment:
Can you use the same pattern with the TestInstance ? and then have consumer as `sut` and verify that `documentStoreService.uploadFile` is called ?